src/openforms/contrib/referentielijsten/api/viewsets.py

```diff
@@ -0,0 +1,55 @@
+from django.shortcuts import get_object_or_404
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from requests.exceptions import RequestException
+from rest_framework import authentication, permissions, serializers, status, viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
+from zgw_consumers.client import build_client
+from zgw_consumers.models import Service
+
+from openforms.api.utils import mark_experimental
+
+from ..client import ReferentielijstenClient
+from .serializers import ReferentielijstTabellenSerializer
+
+
+@extend_schema_view(
+    get=extend_schema(
+        summary=_("List tabellen for a Referentielijsten service"),
+        description=_(
+            "Return a list of available (JSON) tabellen in a given Referentielijsten service configured "
+            "in the backend.\n\n"
+            "Note that this endpoint is **EXPERIMENTAL**."
+        ),
+        responses={
+            200: ReferentielijstTabellenSerializer(many=True),
+        },
+    )
+)
+@mark_experimental
+class ReferentielijstenTabellenViewSet(viewsets.ViewSet):
```

Comment:
Can you use https://open-forms.readthedocs.io/en/stable/developers/backend/core/utils.html#openforms.api.views.mixins.ListMixin like in the rest of the non-model backed list API endpoints?